Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -170,6 +170,13 @@ - (void)track:(SEGTrackPayload *)payload
 
 - (void)screen:(SEGScreenPayload *)payload
 {
+    if ([(NSNumber *)[self.settings objectForKey:@"trackAllPagesV2"] boolValue]) {
+        NSString *event = [[NSString alloc] initWithFormat:@"Loaded a Screen"];
+        [self realTrack:event properties:payload.properties integrations:payload.integrations];
```

Comment:
This method should also add the screen name as an event property so users can still create funnels based on the screen's name.

Reply:
Oh yes, that's key! Also category if that's supported still.

Reply:
Will do!